src/components/forums/forum/ForumContent.tsx

```diff
@@ -43,15 +46,96 @@ import {
 } from '../../../utils/restrictionListHelper';
 import { newPublicKey } from '../../../utils/postbox/validateNewPublicKey';
 import { csvStringToPubkeyList } from '../../../utils/csvStringToPubkeyList';
-import { StarsAlert } from '../StarsAlert';
 
 interface ForumContentProps {
   forumObject: DispatchForum;
-  forumData: ForumData;
+  basicInfo?: { title: string; desc: string };
+  forumData?: ForumData;
   update: () => Promise<void>;
 }
 
 export function ForumContent(props: ForumContentProps): JSX.Element {
+  const { forumData, forumObject, basicInfo, update } = props;
+  const { permission } = forumObject;
+
+  if (isNil(forumData)) {
+    const confirmingBox = (
+      <div className='confirmingBanner'>
+        <div className='title'>
+          <div className='animation'>
+            <Lottie
+              loop
+              animationData={animationData}
+            />
+          </div>
+          <div className='text'>The network is confirming your forum.</div>
+        </div>
+        <div className='subtitle'>When it`s ready, the page will reload itself. This may take a few seconds.</div>
+      </div>
+    );
+
+    if (!isNil(basicInfo)) {
+      const forumHeader = (
+      <div className="forumContentHeader">
+        <div className={'titleBox'}>
+          <Markdown>{basicInfo.title}</Markdown>
+        </div>
+        <div className="descriptionBox">
+          <div className="description">
+            <Markdown>{basicInfo.desc}</Markdown>
+          </div>
+          <button
+            className={'createTopicButton'}
+            type="button"
+            disabled
+            >
+            <div className="buttonImageContainer">
+              <Plus />
+            </div>
+            Create Topic
+          </button>
+        </div>
+      </div>
+      );
+
+      return (
+      <div className='confirmingWrapper'>
+        <div className="forumContent">
+          <>{ReactGA.send('pageview')}</>
+          <div className="forumContentBox" >
+            {!permission.readAndWrite && <ConnectionAlert />}
+            {confirmingBox}
+            {forumHeader}
+          </div>
+          <div className="toolsWrapper" />
+          <div className="topicListWrapper">
+            <TopicList />
+          </div>
+        </div>
+      </div>
+      );
+    } else {
+      return <Spinner/>;
+    }
+  } else {
+    return (
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
see comment about two components in the same file

Reply:
In this case it does make sense to have this in a different component